src/limbo/opt/exhaustive_search.hpp

```diff
@@ -0,0 +1,46 @@
+#ifndef LIMBO_OPT_EXHAUSTIVE_SEARCH_HPP
+#define LIMBO_OPT_EXHAUSTIVE_SEARCH_HPP
+
```

Comment:
I just copied the name from the previous files/declarations. It's true that it seems more for a grid search. Although, grid search should be a kind of an exhaustive search.